### PR TITLE
[bilibili-comics] added website support

### LINKED ISSUE #3402: [Site Request] bilibilicomics
**Name of the website**
bilibilicomics

**Website urls (examples below)**
- Site: https://www.bilibilicomics.com/
- Manga List: https://www.bilibilicomics.com/?from=manga_detail
- Manga example: https://www.bilibilicomics.com/detail/mc55?from=manga_index 
- Chapter online viewer example: https://www.bilibilicomics.com/mc55/321?from=manga_detail

**Languages**
English

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by 27136773**

### CODE REVIEW

File: src/web/mjs/connectors/BilibiliManhua.mjs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                language: ''
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const data = await this._fetchTwirp('/GetImageIndex', {
+            ep_id: chapter.id
+        });
+        let images = data.images.map(image => [ image.path, '@', image.x, 'w.jpg' ].join(''));
```
Comment (Sheepux) [inline src/web/mjs/connectors/BilibiliManhua.mjs]: Question : what's the advantage of the array with join over string concatenation ?

**Review by Sheepux — APPROVED** [review]

File: src/web/mjs/connectors/BilibiliManhua.mjs (hunk 1 above)
Comment (27136773) [reply src/web/mjs/connectors/BilibiliManhua.mjs]: No certain advantage, might be personal preference / readability for certain situations
```javascript
[ image.path, '@', image.x, 'w.jpg' ].join('')
// vs.
image.path + '@' + image.x + 'w.jpg'
// vs.
`${image.path}@${image.x}w.jpg`
```

Comment (Sheepux) [reply src/web/mjs/connectors/BilibiliManhua.mjs]: that felt disturbing as there was a string concatenation 2 lines later :D

Comment (27136773) [reply src/web/mjs/connectors/BilibiliManhua.mjs]: Changed to concatenation

**Pull request merged**